src/main/java/Duke.java

```diff
@@ -1,10 +1,208 @@
+import java.util.ArrayList;
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
```

Comment:
Could split the class into various classes, i.e. TaskList, Event, Deadline, instead of putting them into one class